### PR TITLE
Apply checkstyle in baseapp and proxyserver

### PR DESCRIPTION
**Pull request by ebyhr**

### CODE REVIEW

**Comment by mosabua** [comment]
Please confirm that this commit does not change any functionality and is a pure source reformat and maybe add this info in the commit message @ebyhr 

Thanks for sending the PR btw.. that was our next step ..

**Comment by ebyhr** [comment]
@mosabua Added commit body.

File: baseapp/pom.xml [hunk 1]
```diff
@@ -12,6 +12,10 @@
     <artifactId>baseapp</artifactId>
     <packaging>jar</packaging>
 
+    <properties>
```
Comment (mosabua) [inline baseapp/pom.xml:15]: Great .. this is a good temporary approach until the whole project works.

**Review by mosabua — APPROVED** [review]
Awesome. Thank you so much for taking this on.

**Pull request merged**